packages/react/package.json

```diff
@@ -331,12 +290,12 @@
     "rollup-plugin-filesize": "^9.1.1",
     "rollup-plugin-postcss": "^4.0.0",
     "rollup-plugin-uglify": "^6.0.4",
+    "sass-loader": "^10.0.0",
```

Comment:
Slightly older version of sass-loader, because v11+ requires webpack 5, which is an endeavor in itself..